homework/index.js

```diff
@@ -30,14 +30,93 @@
     return elem;
   }
 
+  function createOption(repositories, select) {
+    createAndAppend('option', select, { text: 'Select a repository', disabled: 'disabled' });
+    repositories
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .forEach(repository => {
+        createAndAppend('option', select, { text: repository.name, value: repository.name });
+      });
+  }
+
+  function renderRepoBasisInfo(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}`;
+    const container = document.getElementById('root');
+    fetchJSON(url, (err, repo) => {
+      if (err) {
+        createAndAppend('div', container, { text: err.message, class: 'alert-error' });
+      }
+      ul.innerHTML = '';
+
+      const li = createAndAppend('li', ul, { class: 'info-list' });
+      const table = createAndAppend('table', li);
+      const tbody = createAndAppend('tbody', table);
+      const trName = createAndAppend('tr', tbody);
+      createAndAppend('th', trName, { text: 'Repository:' });
+      const tdName = createAndAppend('td', trName);
+      createAndAppend('a', tdName, { text: repo.name, href: repo.html_url, target: '_blank' });
+      const trDes = createAndAppend('tr', tbody);
+      createAndAppend('th', trDes, { text: 'Description:' });
+      createAndAppend('td', trDes, { text: repo.description });
+      const trForks = createAndAppend('tr', tbody);
+      createAndAppend('th', trForks, { text: 'Forks:' });
+      createAndAppend('td', trForks, { text: repo.forks_count });
+      const trUpd = createAndAppend('tr', tbody);
+      createAndAppend('th', trUpd, { text: 'Updated at:' });
+      createAndAppend('td', trUpd, { text: repo.updated_at });
+    });
+  }
+
+  function renderContributors(repository, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repository}/contributors`;
+    fetchJSON(url, (err, contributions) => {
+      if (err) return;
+      ul.innerHTML = '<h3>Contributions</h3>';
+      contributions.forEach(contributor => {
+        const li = createAndAppend('li', ul, { class: 'contributor' });
+        const contributorTable = createAndAppend('table', li);
```

Comment:
An HTML `table` is not the best choice here. It is better to use divs in combination with flex-box to position and line up the three components making up the list item.

It would also be nice if you could click anywhere on the list item to go to the contributors page. You can achieve that by wrapping the inner content of the list item in an `a` tag and set its `href` to the appropriate url.